core/src/test/java/co/cask/wrangler/steps/parser/JsParserTest.java

```diff
@@ -142,4 +142,18 @@ public void testFlattenToDepth() throws Exception {
     records = PipelineTest.execute(directives, records);
     Assert.assertTrue(records.size() == 1);
   }
+
+  @Test
+  public void testParsingExtraCharacters() throws Exception {
+    String[] directives = new String[] {
+      "parse-as-json body"
+    };
+
+    List<Record> records = Arrays.asList(
+      new Record("body", "[1,2,3,4,5]             ")
```

Comment:
Because we don't currently have tests for the `FilebrowserExplorer` (or a way to open a file through the `FilebrowserExplorer` through a test, it's difficult to simulate the issue that is seen when parsing the JSON since we just have these unit tests, but this is the sort of input that gets sent to the `JsParser`.